crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1063,6 +1066,9 @@ spec:
                             items:
                               type: object
                               properties:
+                                auto_enable:
+                                  description: "Enabled option by default"
```

Same suggestion as above.

```suggestion
                                auto_select:
                                  description: "If true this option will be selected and take effect automatically. Note that only one option in the list can have this value be set to true."
```